database/seeders/CountySeeder.php

```diff
@@ -0,0 +1,66 @@
+        County::create(["id" => 11, "slug" => "buzau",           "name" => "Buzau"]);
+        County::create(["id" => 12, "slug" => "calarasi",        "name" => "Calarasi"]);
+        County::create(["id" => 13, "slug" => "caras-severin",   "name" => "Caras-Severin"]);
+        County::create(["id" => 14, "slug" => "cluj",            "name" => "Cluj"]);
+        County::create(["id" => 15, "slug" => "constanta",       "name" => "Constanta"]);
+        County::create(["id" => 16, "slug" => "covasna",         "name" => "Covasna"]);
+        County::create(["id" => 17, "slug" => "dambovita",       "name" => "Dambovita"]);
+        County::create(["id" => 18, "slug" => "dolj",            "name" => "Dolj"]);
+        County::create(["id" => 19, "slug" => "galati",          "name" => "Galati"]);
+        County::create(["id" => 20, "slug" => "giurgiu",         "name" => "Giurgiu"]);
+        County::create(["id" => 21, "slug" => "gorj",            "name" => "Gorj"]);
+        County::create(["id" => 22, "slug" => "harghita",        "name" => "Harghita"]);
+        County::create(["id" => 23, "slug" => "hunedoara",       "name" => "Hunedoara"]);
+        County::create(["id" => 24, "slug" => "ialomita",        "name" => "Ialomita"]);
+        County::create(["id" => 25, "slug" => "iasi",            "name" => "Iasi"]);
+        County::create(["id" => 26, "slug" => "ilfov",           "name" => "Ilfov"]);
+        County::create(["id" => 27, "slug" => "maramures",       "name" => "Maramures"]);
+        County::create(["id" => 28, "slug" => "mehedinti",       "name" => "Mehedinti"]);
+        County::create(["id" => 29, "slug" => "mures",           "name" => "Mures"]);
+        County::create(["id" => 30, "slug" => "neamt",           "name" => "Neamt"]);
+        County::create(["id" => 31, "slug" => "olt",             "name" => "Olt"]);
+        County::create(["id" => 32, "slug" => "prahova",         "name" => "Prahova"]);
+        County::create(["id" => 33, "slug" => "salaj",           "name" => "Salaj"]);
+        County::create(["id" => 34, "slug" => "satu Mare",       "name" => "Satu Mare"]);
+        County::create(["id" => 35, "slug" => "sibiu",           "name" => "Sibiu"]);
+        County::create(["id" => 36, "slug" => "suceava",         "name" => "Suceava"]);
+        County::create(["id" => 37, "slug" => "teleorman",       "name" => "Teleorman"]);
+        County::create(["id" => 38, "slug" => "timis",           "name" => "Timis"]);
+        County::create(["id" => 39, "slug" => "tulcea",          "name" => "Tulcea"]);
+        County::create(["id" => 40, "slug" => "valcea",          "name" => "Valcea"]);
+        County::create(["id" => 41, "slug" => "vaslui",          "name" => "Vaslui"]);
+        County::create(["id" => 42, "slug" => "vrancea",         "name" => "Vrancea"]);
+        County::create(["id" => 43, "slug" => "remote",          "name" => "Remote"]);
```

Comment:
Please use one insert in DB with County::insert($array). Also, you can use  this [import](https://github.com/code4romania/rvm/blob/main/database/migrations/2014_10_11_000000_create_siruta_tables.php)

Reply:
@gheorghelupu17 i performed the requested updates